src/pages/settings/Wallet/assignedCardPropTypes.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import CONST from '../../../CONST';
+
+/** Assigned Card props */
+const assignedCardPropTypes = PropTypes.shape({
+    cardID: PropTypes.number,
+    state: PropTypes.number,
+    bank: PropTypes.string,
+    availableSpend: PropTypes.number,
+    domainName: PropTypes.string,
+    maskedPan: PropTypes.string,
+    isVirtual: PropTypes.bool,
+    fraud: PropTypes.oneOf([CONST.EXPENSIFY_CARD.FRAUD_TYPES.DOMAIN, CONST.EXPENSIFY_CARD.FRAUD_TYPES.USER, CONST.EXPENSIFY_CARD.FRAUD_TYPES.NONE]),
+    cardholderFirstName: PropTypes.string,
+    cardholderLastName: PropTypes.string,
+});
+
+export default assignedCardPropTypes;
```

Comment:
the assigned card shape is defined in `card.ts`

Reply:
I don't get the idea 

How do we use card shapes from the card.ts where we have type definition? (prop-types in react component / typescript type)

Reply:
Answered in slack, but I was sending this for myself to double check against ts